src/main/sensors/acceleration.c

```diff
@@ -34,8 +34,15 @@
 
 #include "sensors/acceleration.h"
 
+#include "flight/pid.h"
+#include "flight/filter.h"
+
 int16_t accADC[XYZ_AXIS_COUNT];
 
+static filterState_t accADCState[XYZ_AXIS_COUNT];
+
+static pidProfile_t *pidProfile;
```

Comment:
I don't see place where `pidProfile` is initialized. Without initialization it will be zero - random flash content will be used.